slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -218,7 +218,7 @@ private long validatorId(final Bytes publicKey) {
 
   private void lockForValidator(
       final Handle handle, final LockType lockType, final long validatorId) {
-    final String lockTypePrefix = lockType == LockType.BLOCK ? "0" : "1";
+    final String lockTypePrefix = Integer.toString(lockType.ordinal());
```

Comment:
The problem is that the lockId is NOT guaranteed to fit into a long anymore.

Reply:
validator_id doesn't need to be long so I'll change to be an int then I can use the two int arg version of the lock to avoid any issue with size of the value.